demos/adaptive_circuits_demo.ipynb

```diff
@@ -897,10 +915,14 @@
     "            apply_selected)\n",
     "\n",
     "num_gates = len(doubles)\n",
-    "params = [0.0] * num_gates\n",
-    "gates_verifier = [True] * num_gates\n",
-    "grads = avqe_circuit_grad(num_gates, 1, params, [0.0], doubles, singles,\n",
-    "            [[-1,-1,-1,-1]], [[-1,-1]], gates_verifier, [False], False)\n",
+    "params = jnp.asarray([0.0] * num_gates)\n",
```

Comment:
@maliasadi I'm curious about the change here?

Reply:
We used to convert Python lists to `jax.numpy.array` before tracing in the frontend as JAX doesn't support Python lists. Now, we have support for PyTrees without any implicit list (or array) conversions under the hood. This makes the UI more consistent with JAX's behaivour.